### PR TITLE
Fixes #131 Redirects admin to admin

### LINKED ISSUE #131: [ADMIN]: Redirect from profile page to admin page
**Describe the bug**
We need proper redirects when user is admin

### PR DESCRIPTION
**Pull request by Zjjc123**
Redirects admin to admin when in `/login` and in home page

### CODE REVIEW

**Review by daniel-sudz — DISMISSED** [review]
Something similar for /table would be nice when an admin refreshes the page to go back to the dashboard 

image

**Review by theTisch21 — APPROVED** [review]
I can't test this (I'm not an admin), but it all looks good.

File: website/src/components/table/table.tsx [hunk 1]
```diff
@@ -14,6 +18,16 @@ const CSTable: React.FunctionComponent<{
...
+  const signedInstate = useSelector(selectSignedInState);
+  const history = useHistory();
+
+  useEffect(() => {
+    // only redirect if not viewing another user
+    if (signedInstate.admin && isAdmin(signedInstate.userEmail)) {
+      history.replace("/admin");
+    }
+  });
+
```
Comment (daniel-sudz) [inline website/src/components/table/table.tsx:30]: could you move this to the controller that wraps the table component? The ```table.tsx``` should be a purely UI component

**Review by daniel-sudz — CHANGES_REQUESTED** [review]

**Review by daniel-sudz — APPROVED** [review]
lgtm 👍

**Pull request merged**